colcon_bundle/verb/_overlay_utilities.py

```diff
@@ -80,44 +69,35 @@ def create_dependencies_overlay(staging_path, overlay_path):
     :param str overlay_path: Path of overlay output file
     (.tar.gz)
     """
-    dependencies_staging_path = staging_path
-    dependencies_tar_gz_path = overlay_path
+    dep_staging_path = Path(staging_path)
+    dep_tar_gz_path = Path(overlay_path)
     logger.info('Dependencies changed, updating {}'.format(
-        dependencies_tar_gz_path
+        str(dep_tar_gz_path)
     ))
 
-    _generate_template(
-        'v2_setup.jinja2.sh',
-        'v2_setup.bash',
-        _CONTEXT_VAR_BASH
-    )
+    shellscript_dest = Path(dep_staging_path) / 'setup.sh'
+    shellscript_dest_bash = Path(dep_staging_path) / 'setup.bash'
```

Comment:
nit: put this variable in a visual block with where you use it. not a blocker, just a style preference, e.g.

```
some_tmp_var = x
use(some_tmp_var)
some_tmp_var.something()

some_other_var = y
use(some_other_var)
some_other_var.something()
```